specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -1,6 +1,6 @@
 emit:
-  - "@azure-tools/typespec-autorest"
-  - "@azure-tools/cadl-apiview"
+#  - "@azure-tools/typespec-autorest"
+#  - "@azure-tools/cadl-apiview"
```

Comment:
There is currently no config for `typespec-autorest`, so I don't think it will output any swagger?  Once we add config, should it output to this folder?

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference

Reply:
If this started with a new api-version, that would be the case. As it is, this older api-version is very unlikely to match the swagger in that folder (though it would hopefully be sematically equivalent)

Reply:
This change causes the CI pipeline to "pass", but it's only because no emitter is run:

```
+ pushd specification/cognitiveservices/OpenAI.Inference
+ npx tsp compile .
TypeSpec compiler v0.43.0

Diagnostics were reported during compilation:

warning emitter-not-found: Emitter with name @azure-tools/typespec-csharp is not found.
warning emitter-not-found: Emitter with name @azure-tools/typespec-java is not found.

Found 2 warnings.

No emitter was configured, no output was generated. Use `--emit <emitterName>` to pick emitter or specify it in the typespec config.
```

Do we want specs in the repo with no emitters configured?  What is the scenario for this?

Reply:
It looks like this is here purely for client generation.  If we can agree on where to emit the swagger, it would be good to emit it there

Reply:
I'm fine merging this PR as-is.  Later we will need to decide what to do with specs that are client-only (don't emit swagger).